src/Shared/Model/Enums/JsonParsingOption.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+
+namespace Microsoft.CST.OpenSource.Model.Enums;
+
+/// <summary>
+/// Special cases used for json parsing.
+/// </summary>
+public enum JsonParsingOption
+{
+    Unknown = 0,
```

Comment:
Maybe do None instead of Unknown here since this isn't a value that will be serialized across the wire.

Reply:
Can do!

Reply:
Switched it from unknown to none, and made it so that instead of the option being nullable in `BaseProjectManager` to using `None` instead.